### ISSUE TITLE
Allow opening the "request access to namespace" dialogue directly from a URL

### ISSUE THREAD

**Issue by kolotu** [eclipse/vorto#2462]
The request access to namespace dialogue needs to be accessible directly from a URL.

**Comment by mena-bosch** [comment]
If we want this to be available for non-authenticated users, it will need to strip down part of the functionality in the current form. 
For instance, the search for technical users should be disabled (prevents exposure of usernames to non-authenticated users). 
The search for public namespaces can probably be left as is. 

Another feature that should not be part of the anonymously accessible form in my opinion, is the ability to create a technical user inline (see #2460 - not implemented in the original form as of yet). 

Bottomline, this would reduce the functionality of the anonymous version of the form, in favor of a more conservative approach to anonymous users modifying Vorto data. 

@kolotu thoughts?

**Comment by kolotu** [comment]
@mena-bosch  very good point. Let's go with the authenticated version first - that should be enough to start. We can check at a later point whether the anonymous version is still required.

**Comment by mena-bosch** [comment]
@kolotu fair enough. 
The authenticated version would still have the problem of post-login redirects though. 
In other words, as of now, if punch in a Vorto page that requires authentication and you are not authenticated, you are not redirected to that page past authentication, but to the home page. 

Hopefully this is just some extra work, but I haven't checked feasibility/estimates yet.

**Comment by kolotu** [comment]
@mena-bosch yes that is the challenge for this task - and also a nice feature to have in general, I think.

**Comment by mena-bosch** [comment]
I have decided to split up the "smart redirect" functionality and formalize it in #2470.
Therefore the scope here would only be to provide the "standalone" version of the page.

**Comment by mena-bosch** [comment]
PR [here](https://github.com/eclipse/vorto/pull/2473).

The standalone page can be reached at `[host]:[port]/#/requestAccessToNamespace`.

The code is **very** hack-ish, but I'm running out of time. 

No coverage for now as this is UI-only. 

Seems to work well with #2470, including with a 3 step process such as: 

1. Punch the URL
2. Get redirected to login page
3. Get redirected to GitHub 
4. Get redirected to `requestAccessToNamespace` after authentication successful

**Comment by kolotu** [comment]
Re-opened as the implementation does not work with Bosch ID / Eclipse ID

**Comment by kolotu** [comment]
Also the user ID shall be read from the query parameter "userId"
